src/blueprint/mod.rs

```diff
@@ -4,8 +4,10 @@ mod const_utils;
 mod from_config;
 mod into_schema;
 mod timeout;
+mod validation;
 
 pub use blueprint::*;
 pub use const_utils::*;
 pub use from_config::*;
 pub use timeout::GlobalTimeout;
+pub use validation::*;
```

Comment:
Rename this to `operation`. Because this will contain functions for validating operations.